HARK/abstract/tests/test_variables.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+import numpy as np
+import yaml
+
+import HARK.abstract.variables
+
```

Comment:
A test case showing how this data structure could be used in practice would go a long way.